testsuite/forge/src/backend/k8s/swarm.rs

```diff
@@ -543,35 +585,38 @@ pub fn clean_k8s_cluster(
     });
 
     // healthcheck on each of the validators
-    let unhealthy_validators = validators
-        .iter_mut()
-        .filter_map(|(_, val)| {
-            let val_name = val.name.clone();
-            println!("Attempting health check: {}", val_name);
-            // perform healthcheck with retry, returning unhealthy
-            let check = diem_retrier::retry(k8s_retry_strategy(), || match val.health_check() {
-                Ok(_) => {
-                    println!("Validator {} healthy", val_name);
-                    Ok(())
-                }
-                Err(ref x) => {
-                    println!("Validator {} unhealthy: {}", val_name, x);
-                    Err(())
+    if require_validator_healthcheck {
```

Comment:
I assume we always need to do healthcheck?

Reply:
So the healthcheck only works when you're running forge from within the cluster, since it requires being able to each each of the validator LB services to conduct the node healthchecks. This is to enable operators to run clean up from their laptops (and then we'd verify the health manually)